VMEncryption/test/test_check_util.py

```diff
@@ -345,3 +345,79 @@ def test_supported_os(self):
             }, MockDistroPatcher('SuSE', '12.4', ''), {"os" : "NotEncrypted"})
         except Exception as e:
             self.fail("is_unsupported_os threw unexpected exception.\nException message was:\n" + str(e))
+
+    def test_volume_type_enable(self):
```

Comment:
nit: could be broken into more granular tests, so if one case fails it is easier to pinpoint which one

Reply:
Done.